### PR TITLE
[patch] Fix support for kafka/ocs in mas update  

### PR DESCRIPTION
**Pull request by racree**
while testing mas update cli I found these 2 code changes to be necessary.  
There's an update for kafka provider path and an update to ocs.

### CODE REVIEW

File: ibm/mas_devops/roles/kafka/tasks/main.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
       - "Kafka Action .................... {{ kafka_action }}"
 
 - name: Provision or Deprovision Kafka Instance
-  include_tasks: tasks/provider/{{ kafka_provider }}/{{ kafka_action }}.yml
+  include_tasks: ../roles/kafka/tasks/provider/{{ kafka_provider }}/{{ kafka_action }}.yml
```
Comment (whitfiea) [inline ibm/mas_devops/roles/kafka/tasks/main.yaml:9]: This shouldn't be needed. What was being run when you saw this problem? It sounds like the role path was not set correctly?

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: ibm/mas_devops/roles/kafka/tasks/main.yaml (hunk 1 above)
Comment (racree) [reply ibm/mas_devops/roles/kafka/tasks/main.yaml:9]: I was testing "mas update" and it consistently fails with an invalid path that includes the directory "playbooks". Thus this change prepends "../roles/kafka/".

So there must be an problem with the role path for update in the cli?

Comment (whitfiea) [reply ibm/mas_devops/roles/kafka/tasks/main.yaml:9]: @racree Was this running inside the cli docker container or just running `mas update` directly on your machine? We don't seem to need this change in other places so I suspect something might be changing the path before this.

Comment (racree) [reply ibm/mas_devops/roles/kafka/tasks/main.yaml:9]: I ran the following steps inside the cli docker container.  
Steps:

1. mas install
2. installed kafka using "ROLE_NAME=kafka ansible-playbook ibm.mas_devops.run_role"
3. mas update

mas update consistently failed with a file not found.

Comment (whitfiea) [reply ibm/mas_devops/roles/kafka/tasks/main.yaml:9]: the failure happens in the tekton pipeline, so don't think this environmental

**Review by whitfiea — APPROVED** [review]

**Pull request merged**